### PR TITLE
Tek830 a 16 add interface elements for displaying lamps turning on and off

### PR DESCRIPTION
**Pull request by RezaAmanii**
This pull request contains some new changes:

- Deleted the Lamp.py class instead we have AptLayout.py which represent an apartment class
- Added method to toggle between lighten or darken specific parts of room, like living room, bath room ...

### CODE REVIEW

File: Images/SWITCHOFF.png
Comment (filipkauffeldt) [inline Images/SWITCHOFF.png:1]: Are theses images used?

File: View/AptLayout.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import tkinter as tk
+from PIL import Image, ImageTk, ImageEnhance
+import os
+
+
+class AptLayout:
+    def __init__(self, parent):
+
+        # Darkness intensity
```
Comment (filipkauffeldt) [inline View/AptLayout.py:9]: Are these comments needed? The code seems very self explanatory

**Review by filipkauffeldt — APPROVED** [review]
Looks good

File: Images/SWITCHOFF.png
Comment (RezaAmanii) [reply Images/SWITCHOFF.png:1]: Not now but maybe later when the interface is complete

File: View/AptLayout.py (hunk 1 above)
Comment (RezaAmanii) [reply View/AptLayout.py:9]: Yeah it is ineed needed for someone who does not understand it in the future, if they want to work with it.

**Review by RezaAmanii — COMMENTED** [review]
After reviewing the comments and answering them.

**Pull request merged**